spec/factory_helper/string_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+RSpec.describe FactoryHelper::String do
+  describe '.random' do
+    specify 'is String' do
+      expect(FactoryHelper::String.random).to be_a_kind_of String
+    end
+
+    specify 'is valid' do
+      expect(FactoryHelper::String.random.valid_encoding?).to be_truthy
+    end
+
+    specify 'default encoding' do
+      expect(FactoryHelper::String.random.encoding.to_s).to eq 'UTF-8'
+    end
+
+    specify 'default length' do
+      expect(FactoryHelper::String.random.length).to eq 32
+    end
+
+    describe 'length' do
+      specify 'nil' do
+        expect(FactoryHelper::String.random(nil).length).to eq 0
+      end
+
+      specify 'integer' do
+        length= [0, -1, 1, rand(500), rand(4096)- 2048].sample
+        expect(FactoryHelper::String.random(length).length).to eq [0, length].max
```

Comment:
Consider another name for the local variable just to avoid confusion between it and the method.